### PR TITLE
git setup push

### PR DESCRIPTION
**Pull request by Ramya300**

### CODE REVIEW

File: src/main/java/com/example/todolist/TaskController.java [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+       List<Task> data = taskService.getAllTasks();
+       return ResponseEntity.ok(data);
+    }
+
+    @DeleteMapping("/deleteTask")
+    public ResponseEntity<String> deleteTask(@RequestParam(name = "index") Long id) {
+        try {
+            taskService.deleteTask(id);
+            return ResponseEntity.ok().build();
+        } catch (Exception e) {
```
Comment (govardhankoyaraju) [inline src/main/java/com/example/todolist/TaskController.java:30]: Create Custom Exceptions and controller them with Spring Global Exception handler (https://medium.com/@aedemirsen/spring-boot-global-exception-handler-842d7143cf2a)

File: src/main/java/com/example/todolist/TaskController.java [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+            return new ResponseEntity<>("Failed to delete task: " + e.getMessage(), HttpStatus.INTERNAL_SERVER_ERROR);
+        }
+    }
+
+    @PostMapping("/newTask")
+    public ResponseEntity<String> newTask (@RequestBody String newTask) {
+        try {
+            taskService.addTask(newTask);
+            return ResponseEntity.ok().build();
+        } catch(Exception e){
```
Comment (govardhankoyaraju) [inline src/main/java/com/example/todolist/TaskController.java:40]: Create custom exception.

File: src/main/java/com/example/todolist/TaskController.java (hunk 1 above)
Comment (Ramya300) [reply src/main/java/com/example/todolist/TaskController.java:30]: https://github.com/Ramya300/Task/commit/8ae4b88ed1d5b1de1e0e07902d5d1f58baea49b4

updated with custom exceptions. refer the commit attached.

@govardhankoyaraju

File: src/main/java/com/example/todolist/TaskController.java (hunk 2 above)
Comment (Ramya300) [reply src/main/java/com/example/todolist/TaskController.java:40]: https://github.com/Ramya300/Task/commit/8ae4b88ed1d5b1de1e0e07902d5d1f58baea49b4

Updated.

@govardhankoyaraju

File: src/main/java/com/example/todolist/TaskController.java (hunk 1 above)
Comment (Ramya300) [reply src/main/java/com/example/todolist/TaskController.java:30]: @govardhankoyaraju

File: src/main/java/com/example/todolist/TaskController.java (hunk 2 above)
Comment (Ramya300) [reply src/main/java/com/example/todolist/TaskController.java:40]: https://github.com/Ramya300/Task/pull/4/commits/9ced4f0b48e470ad33bf8fe629bf0570d2cd540f

Updated the naming convension

@govardhankoyaraju

**Pull request merged**